### PR TITLE
CW coordinates validator

### PR DESCRIPTION
**Pull request by aepaysinger**

### CODE REVIEW

**Review by W-Ely — CHANGES_REQUESTED** [review]
See if you can find whats wrong here.

File: code_challenges/code_wars/coordinates_validator.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+def is_valid_coordinates(coordinates):
+    for character in coordinates:
+        if character.isalpha():
+            return False
+    try:
+        x_coordinate = float(coordinates.split(", ")[0])
```
Comment (W-Ely) [inline code_challenges/code_wars/coordinates_validator.py:6]: Only do this split once.

File: code_challenges/code_wars/coordinates_validator.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        y_coordinate = float(coordinates.split(", ")[1])
+    except:
```
Comment (W-Ely) [inline code_challenges/code_wars/coordinates_validator.py:8]: No bare excepts, also write a test to catch whatever case this is handling.

File: code_challenges/code_wars/coordinates_validator.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        return False
+    if len(list(coordinates.split(","))) > 2:
```
Comment (W-Ely) [inline code_challenges/code_wars/coordinates_validator.py:10]: split creates a list, no need to list it.

**Review by W-Ely — CHANGES_REQUESTED** [review]

File: code_challenges/code_wars/coordinates_validator.py (hunk 2 above)
Comment (aepaysinger) [reply code_challenges/code_wars/coordinates_validator.py:9]: i belive i do with my test b

Comment (W-Ely) [reply code_challenges/code_wars/coordinates_validator.py:9]: Great there is a test. Still `No bare excepts,`. You should (almost) never except _all_ exceptions, which one(s) are you expecting, except only it or them.

**Pull request merged**